README.md

```diff
@@ -1,2 +1,82 @@
-# action-tflint
-Run tflint with reviewdog on pull requests to enforce best practices
+# GitHub Action: Run tflint with reviewdog
+
+This action runs [tflint](https://github.com/wata727/tflint) with
+[reviewdog](https://github.com/reviewdog/reviewdog) on pull requests
+to enforce best practices.
+
+## Examples
+
+### With `github-pr-check`
+
+By default, with `reporter: github-pr-check` an annotation is added to
+the line:
+
+![Example comment made by the action, with github-pr-check](./example-github-pr-check.png)
+
+### With `github-pr-review`
+
+With `reporter: github-pr-review` a comment is added to
+the Pull Request Conversation:
+
+![Example comment made by the action, with github-pr-review](./example-github-pr-review.png)
+
+## Inputs
+
+### `github_token`
+
+**Required**. Must be in form of `github_token: ${{ secrets.github_token }}`.
+
+### `level`
+
+Optional. Report level for reviewdog [`info`,`warning`,`error`].
+It's same as `-level` flag of reviewdog.
+The default is `error`.
+
+### `reporter`
+
+Optional. Reporter of reviewdog command [`github-pr-check`,`github-pr-review`].
+The default is `github-pr-check`.
+`github-pr-review` can use Markdown and add a link to rule page in reviewdog reports.
```

Comment:
[nit] It's still not true, so can you remove this sentence?

Reply:
Done!